docs/content/quickstart.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Quickstart Guide
+descriptions: Get started using Porter
+---
+
+## Getting Porter
+
+First make sure Porter is installed.
+Please see the [installation instructions](./install.md) for more info.
```

Comment:
The link should go to `/install/` which will take you to https://porter.sh/install/. You can [preview the doc site locally with Hugo](https://github.com/deislabs/porter/blob/master/CONTRIBUTING.md#documentation) or just rely on the netlify preview.

The reason why I'm asking for the change is that linking to the md files directly (as we are used to for linking within GH) is that Hugo doesn't handle those links properly, and it results in a 404 on the real doc site.